tests/wizard/wizardtests/tests.py

```diff
@@ -391,6 +391,35 @@ def get_context_data(self, **kwargs):
         self.assertEqual(response.context_data['another_key'], 'another_value')
 
 
+class WizardTestPrefix(TestCase):
+    def test_get_prefix(self):
+        class TestWizard(CookieWizardView):
+            def get_prefix(self, request, *args, **kwargs):
+                return 'sample_prefix'
+
+        factory = RequestFactory()
+        view = TestWizard.as_view([forms.Form])
+
+        response = view(factory.get('/'))
+        response.render()
+        self.assertIn(b'sample_prefix', response.content)
```

Comment:
Right, so this `response.context_data['wizard']['management_form'].prefix` contains the prefix. I'll update to check against that instead.